guides/common/modules/proc_installing-the-dhcp-infoblox-module.adoc

```diff
@@ -29,9 +29,13 @@ You cannot use the `host` option without creating conflicts and, for example, be
 --foreman-proxy-plugin-dhcp-infoblox-record-type fixedaddress \
 --foreman-proxy-plugin-dhcp-infoblox-username admin
 ----
-. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}* and select the {SmartProxy} with the Infoblox DHCP module and click *Refresh*.
+. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*, select the {SmartProxy} with the Infoblox DHCP module, and click *Refresh*.
 . Ensure that the *dhcp* features are listed.
 . For all domains managed through Infoblox, ensure that the DNS {SmartProxy} is set for that domain.
 To verify, in the {ProjectWebUI}, navigate to *Infrastructure* > *Domains*, and inspect the settings of each domain.
-. For all subnets managed through Infoblox, ensure that DHCP {SmartProxy} and Reverse DNS {SmartProxy} is set.
+. For all subnets managed through Infoblox, ensure the following settings:
+** The IP address management (IPAM) method of the subnet is set to `DHCP`.
+** DHCP {SmartProxy} and Reverse DNS {SmartProxy} are set.
```

Comment:
The DNS part is "special" here, like above.

I'd reduce this to:
```suggestion
** DHCP Proxy is set to your {SmartProxy}.
```

I don't know if this is branded different in Satellite, but at least in upstream it says "DHCP Proxy", not "DHCP Smart Proxy".

Reply:
Unfortunately, I have to stick with {SmartProxy}, because it is branded in Satellite.